dotnet/AgrirouterTMTProtobufDefinitions/AgrirouterTMTProtobufDefinitions/AgrirouterTMTProtobufDefinitions.csproj

```diff
@@ -12,11 +12,11 @@
         <AssemblyName>agrirouter-tmt-protobuf-definitions</AssemblyName>
         <Product>agrirouter-tmt-protobuf-definitions</Product>
         <RepositoryUrl>https://github.com/DKE-Data/agrirouter-tmt-protobuf-definitions</RepositoryUrl>
-        <PackageVersion>2.1.0</PackageVersion>
+        <PackageVersion>2.2.0</PackageVersion>
```

Comment:
Since the version is changing only within the release process I would recommend not changing the version within the PR.

Reply:
Fixed